VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -460,22 +558,28 @@ public void run() {
     }
 
     /**
+     * callback from EventBus
```

Comment:
Yeah, I copied it from the other `onEvent()` implementations in this class, and neglected changing the javadoc.